admin-core/inc/admin-menu.php

```diff
@@ -436,8 +436,16 @@ public function settings_app_scripts( $localize ) {
 	 *  Add footer link.
 	 */
 	public function add_footer_link() {
-		// translators: HTML entities.
-		return '<span id="footer-thankyou">' . sprintf( __( 'Thank you for using %1$sSpectra.%2$s', 'ultimate-addons-for-gutenberg' ), '<a href="https://wpspectra.com/" class="focus:text-spectra-hover active:text-spectra-hover hover:text-spectra-hover">', '</a>' ) . '</span>';
+		return '<span id="spectra-footer-thankyou" style="font-family: Inter, sans-serif;">' . sprintf(
+			// translators: HTML entities.
+			__(
+				'Enjoyed %1$s? Please leave us a %2$s rating. We really appreciate your support!',
```

Comment:
All the text strings should be in the first parameter of sprintf.
```suggestion
			// translators: %1$s: Opening Strong Tag, %2$s: Closing Strong Tag, %3$s Anchor Tag with Star Symbol Codes
			__(
				'Enjoyed %1$sSpectra%2$s? Please leave us a %3$s rating. We really appreciate your support!',
```